tests/js/handler-caseholder.js

```diff
@@ -0,0 +1,61 @@
+// Test `gpii.schema.middleware` and its subcomponents.
+//
+"use strict";
+var fluid        = fluid || require("infusion");
+var gpii         = fluid.registerNamespace("gpii");
+var jqUnit       = require("jqUnit");
+
+require("../../node_modules/gpii-express/tests/js/lib/test-helpers");
```

Comment:
These will all break in npm 3. Use module-relative includes instead. It looks like no special trickery is required, and this can simply be 
require("gpii-express/tests/js/lib/test-helpers") etc.